core/src/main/java/org/wso2/testgrid/core/command/RunTestPlanCommand.java

```diff
@@ -96,13 +86,15 @@ public void execute() throws CommandExecutionException {
 
             // Get test plan YAML file path location
             Product product = getProduct(productName);
-            Optional<String> testPlanYAMLFilePath = getTestPlanGenFilePath(product);
+            Optional<String> testPlanYAMLFilePath = getTestPlanYamlAbsoluteLocation(product, testPlanConfigLocation);
+            //            Optional<String> testPlanYAMLFilePath = getTestPlanYamlAbsoluteLocation(product);
```

Comment:
Is this kept commented intentionally? :)

Reply:
Code comments should be removed and is considered as a code smell